### DISCUSSION TITLE
Foblex Core is unable to slide the schema

### DISCUSSION THREAD

**Discussion by gakimm** [Foblex/f-flow#36] [Q&A] [answered]
Hello everyone. I have a question.

I'm experiencing an issue where I can't slide the schema. I'm using Node version 18.12.0 and Angular 16.2.0.
it showing error while i need to drag and slide : 
Cannot read properties of undefined (reading 'ctrlKey')
TypeError: Cannot read properties of undefined (reading 'ctrlKey')

Is there a stable version of Foblex/Core compatible with my Node version?

**Comment by siarheihuzarevich** [comment, marked as answer]
Hello, 
Fixed: [Foblex Core is unable to slide the schema https://github.com/Foblex/f-flow/issues/37](https://github.com/Foblex/f-flow/commit/55a407095c39d2428ce3b84d8cd448558dffb993)
Please update flow to v12.5.1 and core to v1.1.8

**Reply by siarheihuzarevich** [reply]
@gakimm Did the solution fix your problem?

**Comment by gakimm** [comment]
Hello Mate!
sorry for late reply
it solved the problem
thank you very much^^

Regards,

Gakimm
